example/project-rails-tailwind/app/components/button_component.html.erb

```diff
@@ -0,0 +1,9 @@
+<button class="<%= classes %>">
+  <div class="flex items-center justify-center">
+    <%= tag.i class: %w[mr-[8px]], data: { feather: "arrow-left" } %>
```

Comment:
In the Figma, these arrow icons looks to be about 10px x 10px instead of 24px x 24px.

Reply:
I changed it to 10.67px as configured in figma but it looks small in comparison. Let me know what you think.

Reply:
Ya it does look a little small. Not a blocker though.